package.json

```diff
@@ -115,7 +115,7 @@
     "unzipper": "^0.10.11",
     "uuid": "^8.3.2",
     "winston": "^3.3.3",
-    "xmldom": "^0.6.0",
+    "xmldom": "github:xmldom/xmldom#0.7.0",
```

Comment:
FYI - this is the workaround required until they can get the library published to npm. Once published to npm we should change this to:
`"xmldom": "^0.7.0"`